src/waveform/renderers/waveformrendererpreroll.h

```diff
@@ -17,7 +17,7 @@ class WaveformRendererPreroll : public WaveformRendererAbstract {
 
   private:
     QColor m_color;
-
+    
```

Comment:
This is a minor nitpick, but please do a quick check for useless whitespace changes before making commits in Git. For this branch it won't really matter, but for bigger branches it makes it harder to review the changes. I do not know what IDE you use, but in KDevelop I use a setting that automatically removes trailing whitespaces when I save files, so if there's a line that just has an indentation like this, it would remove the superfluous spaces.

Reply:
ok, I will see if such option exists in VSCode.

Reply:
https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut